app/controllers/api_controller.rb

```diff
@@ -122,11 +122,11 @@ def set_access_control_headers
   end
 
   def base_config
-    Api::Settings.base
+    ManageIQ::API::Settings.base
```

Comment:
Once the `ApiController` gets namespaced, wil we be removing this verbosity with the full namespace since they'd share some of the same namespace?